x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/Distribution/index.tsx

```diff
@@ -79,7 +81,16 @@ export function ErrorDistribution({ distribution, title }: Props) {
       <Histogram
         tooltipHeader={tooltipHeader}
         verticalLineHover={(bucket: FormattedBucket) => bucket.x}
-        xType="time"
+        xType="time-utc"
+        formatX={(value: any) => {
+          if (value && 'getTime' in value) {
+            const time = value.getTime();
+            return scaleUtc().tickFormat()(
```

Comment:
in `public/components/shared/charts/CustomPlot/StaticPlot.js`, we also use .domain(), is that not necessary/possible here? any difference?

Reply:
Let me make some tests with more than 24h because in the documentation it says `"The default domain is one full day"`...